administration/src/bp-modules/components/CustomDatePicker.tsx

```diff
@@ -0,0 +1,59 @@
+import { DesktopDatePicker } from '@mui/x-date-pickers'
+import { deDE } from '@mui/x-date-pickers/locales'
+import React, { ReactElement } from 'react'
+
+import useWindowDimensions from '../../hooks/useWindowDimensions'
+
+type CustomDatePickerProps = {
+  date: Date | null
+  onChange: (date: Date | null) => void
+  onClear: () => void
+  isValid: boolean
+  minDate?: Date
+  maxDate?: Date
+  disableFuture: boolean
+}
+
+const CustomDatePicker = ({
+  date,
+  onChange,
+  onClear,
+  isValid,
+  minDate,
+  maxDate,
+  disableFuture,
+}: CustomDatePickerProps): ReactElement => {
+  const { viewportSmall } = useWindowDimensions()
+  const formStyle = viewportSmall ? { fontSize: 16, padding: '9px 10px' } : { fontSize: 14, padding: '6px 10px' }
+  const textFieldBoxShadow =
+    '0 0 0 0 rgba(205, 66, 70, 0), 0 0 0 0 rgba(205, 66, 70, 0), inset 0 0 0 1px #cd4246, inset 0 0 0 1px rgba(17, 20, 24, 0.2), inset 0 1px 1px rgba(17, 20, 24, 0.3)'
```

Comment:
just copy the box shadow from the standard TextField components of MUI.
If someone know how to improve that, just let me know

Reply:
thx for the hint. i even don't get this particular styling.
I think we should unify the form fields to mui to have the same styling. currently the text input is blueprint and the date fields are mui. I will create a ticket for that